controllers/auth.controller.js

```diff
@@ -0,0 +1,88 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs')
+const constant = require('../utils/constant');
+const user = require('../models/user.model');
+const config = require('../configs/auth.config')
+
+
+exports.signup = async (req,res) => {
+
+   
+
+    const userObjToBeStored ={
```

Comment:
For the doctor to register, hospital id should be provided.. This check is missing